### PR TITLE
Feat: get all pokemons from API

### PR DESCRIPTION
**Pull request by naftalima**
## Objective and Motivation
- Using Celery Beat to download all the Pokemon at a specific day of the week.

### CODE REVIEW

File: backend/pokebattle/settings/base.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def base_dir_join(*args):
     "django.contrib.messages",
     "django.contrib.staticfiles",
     "django_js_reverse",
+    "django_celery_beat",
```
Comment (rsarai) [inline backend/pokebattle/settings/base.py:46]: Why are you using this library?

**Review by rsarai — CHANGES_REQUESTED** [review]
Periodic tasks should be added in the following way: https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#introduction. I recommend you to set the periodic tasks manually since it's the way we do it on our projects

**Comment by rsarai** [comment]
> Periodic tasks should be added in the following way: https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#introduction. I recommend you to set the periodic tasks manually since it's the way we do it on our projects

This should be made by adding the tasks on `celerybeat_schedule.py`

**Comment by naftalima** [comment]
TODO:  'offset' to move to the next page
Add a 'limit' query parameter to the GET request: ?limit=60&offset=889

**Pull request closed without merging**